Wikipedia/Code/Article as a Living Document/ArticleAsLivingDocLargeEventCollectionViewCell.swift

```diff
@@ -270,12 +289,26 @@ class ArticleAsLivingDocLargeEventCollectionViewCell: CollectionViewCell {
         case .thankAndViewChanges:
             contentView.addSubview(thankButton)
             contentView.addSubview(viewChangesButton)
+            
+            thankButton.titleLabel?.font = UIFont.wmf_font(.body, compatibleWithTraitCollection: traitCollection)
+            viewChangesButton.titleLabel?.font = UIFont.wmf_font(.body, compatibleWithTraitCollection: traitCollection)
+
+            if largeEvent.wereThanksSent {
+                thankButton.setImage(UIImage(named: "thank")?.withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
What do you think about removing the `alwaysTemplate` rendering mode here and instead specifying that mode as the render type in the `Images` asset catalog itself? It seems like in general that's how we're handling the majority of template based images in the app.

Reply:
Updated, thanks.